src/App.js

```diff
@@ -1,24 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
```

Comment:
You successfully created a single page application (SPA) using react-router-dom to serve each page without reloading 😎